components/core/CMakeLists.txt

```diff
@@ -105,37 +105,72 @@ else()
     message(FATAL_ERROR "Could not find ${CLP_LIBS_STRING} libraries for Boost")
 endif()
 
-# Find and setup fmt
-# NOTE:
-# - We only try to link to the static library
-# - spdlog uses fmt, so their versions need to be kept in-sync
-find_package(fmt 8.0.1 REQUIRED)
-if(fmt_FOUND)
-    message(STATUS "Found fmt ${fmt_VERSION}")
-else()
-    message(FATAL_ERROR "Could not find static libraries for fmt")
-endif()
+# Find and setup spdlog and fmt transitive dependency
 
-# Find and setup spdlog
+# Specify the minimum required spdlog version (can be overridden by the user)
+set (spdlog_MIN_VERSION "1.9.0" CACHE STRING "Minimum required version of spdlog")
```

Comment:
This should be `CLP_SPDLOG_MIN_VERSION` according to our [CMake guidelines](https://www.notion.so/yscope/WIP-Coding-Guidelines-9a308b847a5343958ba3cb97a850be66?pvs=4#15c04e4d9e6b80ad8f19e8bdb07cdca0).